### PR TITLE
add getText()/setText() methods to button

### LINKED ISSUE #418: Add setText() and getText() methods to button.
It is useful to have such methods directly in `button` block.

### PR DESCRIPTION
**Pull request by persidskiy**
Close #418

@dfilatov @narqo

### CODE REVIEW

File: common.blocks/button/button.spec.js [hunk 1]
```diff
@@ -140,6 +145,22 @@ describe('button', function() {
...
+            button.getText(1).should.be.equal('world');
+        });
+
+        it('should update text in the text element by index', function() {
+            button.setText('hi');
+            button.elem('text').eq(0).text().should.be.equal('hi');
+
+            button.setText('universe', 1);
+            button.elem('text').eq(1).text().should.be.equal('universe');
+        });
```
Comment (persidskiy) [inline common.blocks/button/button.spec.js:152]: Should I add tests for html-escaping here?

**Comment by dfilatov** [comment]
Why do we need to have multiple text element?

File: common.blocks/button/button.js [hunk 2]
```diff
@@ -57,6 +57,31 @@ provide(BEMDOM.decl({ block : this.name, baseBlock : BaseControl }, /** @lends b
...
+     * @returns {String}
+     */
+    getText : function(index) {
+        return this._getTextByIndex(index).text();
+    },
+
+    /**
+     * Sets text to the `text` element
+     * @param {String} text
+     * @param {Number} [index=0] index of the text element
```
Comment (apostololeg) [inline common.blocks/button/button.js]: слишком уж низкоуровнево, как для внешнего API

**Comment by persidskiy** [comment]
Hmm... you asked a good question. I saw the [examples](https://github.com/bem/bem-components/blob/v2/common.blocks/button/button.tests/simple.bemjson.js#L173-179) of using multiple `text` elems and decided it can be useful, but I could not find any real cases of using it.

I will get rid of these indexes :)

File: common.blocks/button/button.js (hunk 2 above)
Comment (persidskiy) [inline common.blocks/button/button.js]: Да, после поиска примеров использования, кажется, что уже и не надо :)

**Comment by persidskiy** [comment]
:up:

**Comment by dfilatov** [comment]
LGTM

File: common.blocks/button/button.js [hunk 3]
```diff
@@ -54,6 +54,25 @@ provide(BEMDOM.decl({ block : this.name, baseBlock : BaseControl }, /** @lends b
         }
     },
 
+    /**
+     * Returns text form the text element
```
Comment (narqo) [inline common.blocks/button/button.js]: "form" (sic)?

File: common.blocks/button/button.spec.js [hunk 4]
```diff
@@ -140,6 +140,17 @@ describe('button', function() {
             spy.should.not.have.been.called;
         });
     });
+
+    describe('setText/getText', function() {
+        it('should return text from the text element by index', function() {
```
Comment (narqo) [inline common.blocks/button/button.spec.js]: guess, "by index" part is obsolete

File: common.blocks/button/button.js (hunk 3 above)
Comment (dfilatov) [inline common.blocks/button/button.js]: I think we should avoid using inner details in jsdoc.

**Comment by persidskiy** [comment]
:up:

**Pull request merged**